### ISSUE TITLE
Redefine contributors/maintainers/authors field in package sidebar

### ISSUE THREAD

**Issue by ayusharma** [verdaccio/verdaccio#853]
#### My reason:
There is no `contributors` field in current packages on NPM repositories.
#### Steps to reproduce:
```
yarn info react | grep contributors
```
```
yarn info jquery | grep contributors
```
```
yarn info vue | grep contributors
```
#### App Version:
All versions.
#### Config file:
Not needed
#### Additional information:
Not needed
#### Additional verbose log:
Not needed

**Comment by MasterOdin** [comment]
So I found this issue as I was going to file one about how you could publish a package that has a string set for the contributors field in package.json and it'd cause Verdaccio to crash when attempting to view that package. However, this does raise some questions for me on what exactly this issue is trying to suggest.

My use case is that we need someway to list the people that are actively working on a project within the group so that if someone using it has a question, they know who to contact/email. From the [npm package.json documentation](https://docs.npmjs.com/files/package.json), it seems like the [contributor field](https://docs.npmjs.com/files/package.json#people-fields-author-contributors) fits that bill and we've been using that and it also works well within Verdaccio. It seems that this issue wants to going forward ignore that field and replace it with a verdaccio specific field within the package.json called "maintainers" and use that to populate the sidebar on the webpage?

"Maintainers" is a specific thing though to NPM wherein it's a way to specify who can publish a package. Some number of packages does have "contributors", but it's not clear what that means as some are blank and some are not and I have no idea why that it is:

```
C:\Users\mpeveler>npm info verdaccio contributors

[ '030 <chocolatey030@gmail.com>',
  'Aaron Lozier <aaron@potenzainnovations.com>',
  'Adam Szmyd <szmydadam@gmail.com>',
  ...
  'trent.earl <trent.earl@malauzai.com>',
  'verdacciobot <35213902+verdacciobot@users.noreply.github.com>',
  '丸子 <princeb4d@gmail.com>' ]
```

**Comment by ayusharma** [comment]
>  verdaccio specific field within the package.json called "maintainers" and use that to populate the sidebar on the webpage?

It is not a verdaccio related field, NPM returns a list of maintainers who published the package. 

> Some number of packages does have "contributors", but it's not clear what that means as some are blank and some are not and I have no idea why that it is

I am also in the same thought. Officially NPM has not mentioned anything about deprecation in the docs. Also, most of the packages do not have `conttibutor`'s field. Even jquery (it is very old package.)
```
yarn info jquery | grep 'contributors'
```
or 
```
npm info jquery contributors
```

**Comment by MasterOdin** [comment]
> It is not a verdaccio related field, NPM returns a list of maintainers who published the package.

And they do this with an internal field specific to modules, not by utilizing anything in the package.json (except maybe taking author field on the initial publish). 

It's still not clear to me what your plans are for replacing the contributor field though within verdaccio, or exactly what the plan is to list the people who are working on a given module. Based on #900, it sounds like you guys are going to allow usage of a non-official [NPM package.json field](https://docs.npmjs.com/files/package.json), maintainers, but which also maybe weakly maps to what that internally means for NPM packages?

I also wanted to express support for keeping the contributors field as it's something my group has been using for a while now.

**Comment by ayusharma** [comment]
@MasterOdin We have not started on working this yet. It is in `discuss` phase.  

Currently, we skip maintainers and show only `authors` & `contributors`. 

> I also wanted to express support for keeping the contributors field as it's something my group has been using for a while now.

Thanks 😃 Your point is noted.

**Comment by juanpicado** [comment]
I think @MasterOdin made a good point. I guess the depreciation cannot proceed and rather we have to review the process of how we handle the difference between `maintainers` and `contributors` field into the local cache.

**Comment by ayusharma** [comment]
@juanpicado @MasterOdin I agree bo with of you. I am thinking to re-define/ revamp the package sidebar with different sections of `maintainers`, `author` and `contributors`.

**Comment by ayusharma** [comment]
We'll show contributors, maintainers, authors in 3 different sections. 

cc @juanpicado @priscilawebdev

**Comment by juanpicado** [comment]
Solved on #1195
